### PR TITLE
Feature/authorization authentication

### PR DESCRIPTION
**Pull request by DandyCV**
Add Devise & Pundit gems to Gemfile. Add Devise User model. Add Devise default views. Add Home page and controller. Set default generated test files to rspec test.

### CODE REVIEW

File: .idea/Ch-110.iml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    <orderEntry type="library" scope="PROVIDED" name="webpacker (v4.3.0, ruby-2.7.2-p137) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="websocket-driver (v0.7.3, ruby-2.7.2-p137) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="websocket-extensions (v0.1.5, ruby-2.7.2-p137) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="xpath (v3.2.0, ruby-2.7.2-p137) [gem]" level="application" />
+  </component>
+  <component name="RModuleSettingsStorage">
+    <LOAD_PATH number="0" />
+    <I18N_FOLDERS number="1" string0="$MODULE_DIR$/config/locales" />
+  </component>
+</module>
```
Comment (ghost) [inline .idea/Ch-110.iml:101]: Add new line at end of file.

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>Here is home page!</h1>
+
+<% if current_user %>
+  <%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
+<% else %>
+<%= link_to 'Sign up', new_user_registration_path %>
+
+<%= link_to 'Sign in', new_user_session_path %>
+<% end %>
```
Comment (ghost) [inline app/views/home/index.html.erb]: Add new line at end of file.

**Review by ghost — CHANGES_REQUESTED** [review]

File: .idea/Ch-110.iml (hunk 1 above)
Comment (DandyCV) [reply .idea/Ch-110.iml:101]: This is .iml file

File: app/views/home/index.html.erb (hunk 2 above)
Comment (DandyCV) [reply app/views/home/index.html.erb]: Done.

**Pull request merged**